### ISSUE TITLE
Training on GTX 1080 Ti

### ISSUE THREAD

**Issue by tekeburak** [jkjung-avt/hand-detection-tutorial#40]
Hi. Thank you for awesome work.
I have invoked `train.sh`, it seems to run normal but when I checked the GPU memory usage using `nvidia-smi`, 141MB memory has being used. I have increased the `batch_size` in `ssd_mobilenet_v2_egohands.config` file from 24 to 128 but still the model using 141MB memory during training.
What could I do in order to train the model faster?

**Comment by jkjung-avt** [comment]
TensorFlow would pre-allocate GPU memory when it initializes itself, so you probably cannot see the real amount of GPU memory usage by "nvidia-smi".

In general, you could train the model faster by:

* using larger batches and less iterations,
* applying larger learning rates in early part of the training (might need to warm up with small learning rates first).

You should look at the loss curve closely to determine whether you have set the hyperparameters and iterations properly.

**Comment by tekeburak** [comment]
Thank you for your answer @jkjung-avt. I have set the batch size larger but training is really slow.
I'm afraid that it is being trained on CPU rather than GPU because almost 10 hours but still training.
This is the prompt when I started the training.

```
2021-01-26 09:14:42.632676: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:    name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582 pciBusID: 0000:04:00.0`
```
I think that the problem related to CUDA version mismatch. Look at the prompt. I use `tensorflow-gpu 1.15.0` and `CUDA 11.0` is not supported.

```
2021-01-26 09:14:42.632882: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcudart.so.10.0'; dlerror: libcudart.so.10.0: cannot open shared object file: No such file or directory; LD_LIBRA$Y_PATH: /usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.632944: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcublas.so.10.0'; dlerror: libcublas.so.10.0: cannot open shared object file: No such file or directory; LD_LIBRA$Y_PATH: /usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.633000: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcufft.so.10.0'; dlerror: libcufft.so.10.0: cannot open shared object file: No such file or directory; LD_LIBRARY$PATH: /usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.633066: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcurand.so.10.0'; dlerror: libcurand.so.10.0: cannot open shared object file: No such file or directory; LD_LIBRA$Y_PATH: /usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.633121: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcusolver.so.10.0'; dlerror: libcusolver.so.10.0: cannot open shared object file: No such file or directory; LD_L$BRARY_PATH: /usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.633187: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcusparse.so.10.0'; dlerror: libcusparse.so.10.0: cannot open shared object file: No such file or directory; LD_L$BRARY_PATH: /usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.633244: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcudnn.so.7'; dlerror: libcudnn.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH:
/usr/local/cuda-11.0/lib64 :/usr/local/cuda-11.0/lib64

2021-01-26 09:14:42.633259: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1641] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
```

I will create a `Dockerfile` and train the model again. Let's see if using docker will overcome the problem.
